extensions/amp-analytics/analytics-vars.md

```diff
@@ -0,0 +1,105 @@
+# <a name="amp-analytics"></a> Variables supported in `amp-analytics`
+
+## Using the `vars`
+
+The format `${var-name}` can be used when a page or platform defined variable needs to be used in a request string. The `amp-analytics` tag will replace the template with the actual value at the time of construction of the analytics request. All the values substituted using this method are first url encoded using `encodeUrlComponent`. For example:
+
+```html
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
+    ...
+  },
+  "vars": {
+    "account": "ABC123"
+  },
+  "triggers": {
+    ...
+  }
+}
+</script>
+</amp-analytics>
+```
+
+## Supported Vars
+
```

Comment:
Re-mentioning comment above to get rid of this section and instead move up the actual vars into h2s.